charts/linkerd2/values.yaml

```diff
@@ -182,10 +185,10 @@ proxyInjector:
   # Namespace selector used by admission webhook
   namespaceSelector:
     matchExpressions:
-    - key: config.linkerd.io/admission-webhooks
-      operator: NotIn
-      values:
-      - disabled
+      - key: config.linkerd.io/admission-webhooks
```

Comment:
This extra indentation for lists seems unrelated to this change and also is inconsistent with how yaml lists are formatted elsewhere.

Reply:
ugh vscode